elevation_mapping/include/elevation_mapping/sensor_processors/PerfectSensorProcessor.hpp

```diff
@@ -25,28 +25,28 @@ class PerfectSensorProcessor : public SensorProcessorBase
    * @param nodeHandle the ROS node handle.
    * @param transformListener the ROS transform listener.
    */
-  PerfectSensorProcessor(ros::NodeHandle& nodeHandle, tf::TransformListener& transformListener);
+   PerfectSensorProcessor(ros::NodeHandle& nodeHandle, tf::TransformListener& transformListener);
```

Comment:
This should be 2 whitespaces, referring to [this styleguide](https://leggedrobotics.github.io/styleguide/cppguide.html#Spaces_vs._Tabs)